workflow/scripts/build_base_network.py

```diff
@@ -796,15 +819,14 @@ def main(snakemake):
     lines_gis.to_csv(snakemake.output.lines_gis)
 
     # add snapshots
-    snapshot_config = snakemake.params["snapshots"]
-    n.set_snapshots(
-        pd.date_range(
-            freq="h",
-            start=pd.to_datetime(snapshot_config["start"]),
-            end=pd.to_datetime(snapshot_config["end"]),
-            inclusive=snapshot_config["inclusive"],
-        ),
-    )
+    sns_config = snakemake.params["snapshots"]
+    planning_horizons = snakemake.params.planning_horizons
+    sns = make_snapshots(sns_config, planning_horizons)
+    n.snapshots = sns
+    if planning_horizons:
+        n.set_investment_periods(periods=planning_horizons)
+    else:
+        n.set_investment_periods(periods=n.snapshots[0].year)
```

Comment:
When would panning_horizons ever be None here?

Reply:
For validation runs, I believe! In that case, your planning horizon is empty and it just takes the snapshot year as the planning horizon. Justification for this was it is easier to manage always having multi-index, rather than separate logic for single vs. multi-index

Reply:
OOoo okay- for validation runs, I have been setting planning horizon to 2019 so either way works.

Reply:
Fair point! have updated to require planning horizons. Porbably easier in the long run as well, if we alwasys know at least one planning horizon must be provided.